std_msgs/Header header
std_msgs/Int32 id
std_msgs/String class_id